inst/include/Rcpp/r_cast.h

```diff
@@ -67,9 +67,17 @@ namespace Rcpp {
             default:
                 const char* fmt = "Not compatible with requested type: "
                                   "[type=%s; target=%s].";
+#ifndef NDEBUG
+                fprintf(stderr,
+                        fmt,
+                        Rf_type2char(TYPEOF(x)),
+                        Rf_type2char(RTYPE));
+                abort();
+#elif
```

Comment:
#else instead of #elif?

Reply:
Yes... whoops. But it worked with clang, so I missed it.